src/test/java/com/pontusvision/gdpr/PVBasicTest.java

```diff
@@ -296,64 +248,55 @@ public void test00004SharepointDataSources(String jsonFile, String ruleName, Str
 
   }
 
-  @Test
-  public void test00005TotvsPloomes() throws InterruptedException {
```

Comment:
Why was this removed?  Please don't remove tests from PVBasic Test, as they could have an impact on all other subsequent tests.